x-pack/plugins/canvas/public/components/workpad_header/editor_menu/editor_menu.component.tsx

```diff
@@ -155,11 +153,7 @@ export const EditorMenu: FC<Props> = ({
         <EuiContextMenu
           initialPanelId={0}
           panels={editorMenuPanels}
-          className={`canvasSolutionToolbar__editorContextMenu ${
-            isDarkThemeEnabled
-              ? 'canvasSolutionToolbar__editorContextMenu--dark'
-              : 'canvasSolutionToolbar__editorContextMenu--light'
-          }`}
+          className={`canvasSolutionToolbar__editorContextMenu`}
```

Comment:
Is class canvasSolutionToolbar__editorContextMenu still defined? I could not find this defined in any scss file in x-pack/plugins

Reply:
It doesn't like we're using it for any CSS. I removed this class as well